builder/Dockerfile.linux

```diff
@@ -18,9 +18,22 @@ COPY . .
 
 RUN go build ./cmd/aks-periscope
 
+# Add dependencies for building nsenter
+RUN apt-get update && \
+    apt-get install -y autoconf autopoint bison gettext libtool
+
+# Create a statically-compiled nsenter binary (see: https://github.com/alexei-led/nsenter/blob/master/Dockerfile)
+# nsenter versions: https://www.kernel.org/pub/linux/utils/util-linux/
+ADD https://github.com/util-linux/util-linux/archive/v2.38.tar.gz .
```

Comment:
💡☕️:+1: Seems like a sane choice, even though this was released last year.

Reply:
It looked to me as if there was only one more recent release, 2.39, but that was still at RC stage... Anyway, tested and it seems to work!